cfe_internal/enterprise/ha/ha_update.cf

```diff
@@ -0,0 +1,141 @@
+bundle agent ha_update
+{
+ vars:
+      "master_hub_location" string => "/var/cfengine/state/master_hub.dat";
+      "hubs_keys_location" string => "/var/cfengine/ppkeys_hubs";
+      "ppkeys_staging" string => "/var/cfengine/ppkeys_staging";
```

Comment:
Makes sense, but this stuff is used in server bodies. I can try to experiment with common and server bodies to make sure it will work as expected.